confidant/public/modules/resources/controllers/CredentialDetailsCtrl.js

```diff
@@ -61,14 +61,14 @@
                 Credential.get({'id': $stateParams.credentialId}).$promise.then(function(credential) {
                     var _credentialPairs = [],
                         _metadata = [];
-                    if (credential.credential_pairs) {
+                    if (Object.entries(credential.credential_pairs).length) {
```

Comment:
```suggestion
                    if (angular.equals({}, credential.credential_pairs)) {
```

Reply:
This is the consistent way to check this used elsewhere in the angular code.

Reply:
ha awesome.  was hoping there'd be a better way.

Small change, I flipped it to `not`, ie. `if (!angular.equals({}, credential.credential_pairs)`